### PR TITLE
Strings deep dive

### PR DESCRIPTION
**Pull request by tong-lambda**
Added string methods tests

### CODE REVIEW

**Review by josh-egan — COMMENTED** [review]
Looks great! :clap: Couple small suggestions below.

File: test/strings-deep-dive.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
+const assert = require("assert");
+
+const sentence = "The quick brown dog Jumps over the lazy dog.";
+
+describe("Strings deep dive", function () {
+  it("Should return charAt(6)", function () {
+    assert.strictEqual(sentence.charAt(6), "i");
+  });
+  it("Should return indexOf('dog')", function () {
```
[the remark this answers is not in the document]
Comment (tong-lambda) [reply test/strings-deep-dive.js:9]: fixed the string

[the remark this answers is not in the document]
Comment (tong-lambda) [reply test/strings-deep-dive.js:9]: added the `cat` test

**Review by josh-egan — APPROVED** [review]
:muscle: :ship:

**Pull request merged**